spec/integrations/factory_default_spec.rb

```diff
@@ -6,6 +6,6 @@
   specify "RSpec integration", :aggregate_failures do
     output = run_rspec('factory_default')
 
-    expect(output).to include("5 examples, 0 failures")
+    expect(output).to include(" examples, 0 failures")
```

Comment:
?

Reply:
here we are interested more in the spec passing, thought it is redundant to update test count each time the fixture spec has a test added, restored to explicit count